contrib/templates/data-engineering/assets/etl-pipeline/template/assets/{{.pipeline_name}}/explorations/README.md

```diff
@@ -0,0 +1,4 @@
+# explorations
+
+This folder is reserved for personal, exploratory notebooks.
+By default these are not committed to Git, as 'explorations' is listed in .gitignore.
```

Comment:
This folder is not gitignored right now.

Reply:
How so? Gitignore says

```
explorations/**
!explorations/README.md
```

Reply:
I see, looks like it's missing somehow. Added. It's in `templates/data-engineering/template/{{.project_name}}/.gitignore`.